keyboards/handwired/asesmero/2x6/keymaps/default/readme.md

```diff
@@ -0,0 +1,14 @@
+# Default Asesmero 2x6 Layout
+
+A simple layout with 12 keys with shorcuts to assign on OBS or any software
+
+LAYER 0 
++------------+------------+------------+------------+------------+------------+
+|    Ctrl    |    Ctrl    |    Ctrl    |    Ctrl    |    Ctrl    |    Ctrl    |
+| + Shift    | + Shift    | + Shift    | + Shift    | + Shift    | + Shift    |
+| + Alt + A  | + Alt + B  | + Alt + C  | + Alt + D  | + Alt + E  | + Alt + F  |
++------------+------------+------------+------------+------------+------------+
+|    Ctrl    |    Ctrl    |    Ctrl    |    Ctrl    |    Ctrl    |    Ctrl    |
+| + Shift    | + Shift    | + Shift    | + Shift    | + Shift    | + Shift    |
+| + Alt + G  | + Alt + H  | + Alt + I  | + Alt + J  | + Alt + K  | + Alt + L  |
++------------+------------+------------+------------+------------+------------+
```

Comment:
You should insert a triple backtick sequence \`\`\` on Line 4, and then again on a new line at the end of the file, and then your keymap drawing will render as intended.

Current code renders like:
![image](https://github.com/qmk/qmk_firmware/assets/18669334/5dd25fe3-bc93-434c-a3fb-9fd950475240)